### DISCUSSION TITLE
Live Logging Start at Midnight

### DISCUSSION THREAD

**Discussion by oralallen82** [jasonacox/Powerwall-Dashboard#41] [Ideas]
Is there a way to have the live logging run a 24hrs period starting at Midnight? That way the estimated savings will match my utility billing period.

**Comment by jasonacox** [comment]
Hi @oralallen82  - yes, you can select a custom range or the macro "Today" in the duration dropdown on the top right:

<img width="219" alt="image" src="https://user-images.githubusercontent.com/836718/175785542-7fdd7c92-eed6-4e20-a912-d8fd6f448057.png">

Click "Save" and it will be your default view.

Example:
<img width="852" alt="image" src="https://user-images.githubusercontent.com/836718/175785657-256d201a-1349-4437-bfb4-191f9595106e.png">

**Reply by oralallen82** [reply]
I totally missed that. Thanks for pointing it out. I should have scrolled down a bit more. Great work again and thanks.
